v2/internal/app/app_dev.go

```diff
@@ -73,12 +73,18 @@ func CreateApp(appoptions *options.App) (*App, error) {
 	var devServerFlag *string
 	var frontendDevServerURLFlag *string
 	var loglevelFlag *string
+	var obfuscateFlag *bool
 
 	assetdir := os.Getenv("assetdir")
 	if assetdir == "" {
 		assetdirFlag = devFlags.String("assetdir", "", "Directory to serve assets")
 	}
 
+	obfuscate := os.Getenv("WAILS_OBFUSCATE")
+	if obfuscate == "" {
+		obfuscateFlag = devFlags.Bool("obfuscate", false, "Directory to serve assets")
```

Comment:
The description seems to be wrong:
```suggestion
		obfuscateFlag = devFlags.Bool("obfuscate", false, "Code obfuscation of bound Wails methods")
```